containenv/containenv/cli/commands/init.py

```diff
@@ -48,3 +77,152 @@ def __call__(self):
         return run_tasks(
             'running some tasks',
             self.tasks)
+
+    def _make_task(self, method):
+        '''create a run_tasks compliant task'''
+        return (method.__doc__, method)
+
+    def make_path(self):
+        '''construct target project path'''
+        self.proj_path = os.path.abspath(self.metasource)
+        self.proj_name = os.path.basename(self.proj_path)
+
+    def _check_project_existence(self):
+        '''verify that self.proj_path is a valid project'''
+        if not os.path.isdir(self.proj_path):
```

Comment:
If the User passes in an invalid target (not a directory) I want to handle/report it ASAP.   Is this OK?   @dangle I think we need to discuss this a little more.